examples/stage-2-shield-study/addon/bootstrap.js

```diff
@@ -126,6 +131,7 @@ async function chooseVariation() {
   return toSet;
 }
 
+// TODO marcrowo: Move to lib?
```

Comment:
Should I maintain this functionality (ie. in `beginStudy` or `endStudy` functions)?

In [`Config.jsm`](https://github.com/marcrowo/shield-studies-addon-utils/blob/066e70b5c2a09f2210d996d4c92b9c2774fcdb1c/examples/stage-2-shield-study/addon/Config.jsm#L70) it is possible to specify modules that are to be imported, but I'm not sure how important this is.